lib/features/children/overview/widgets/child_item.dart

```diff
@@ -1,130 +1,145 @@
 import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:givt_app/app/routes/routes.dart';
+import 'package:givt_app/core/enums/enums.dart';
+import 'package:givt_app/features/auth/cubit/auth_cubit.dart';
+import 'package:givt_app/features/children/overview/models/profile.dart';
 import 'package:givt_app/l10n/l10n.dart';
-import 'package:givt_app/utils/app_theme.dart';
+import 'package:givt_app/utils/utils.dart';
+import 'package:go_router/go_router.dart';
+import 'package:intl/intl.dart';
 
 class ChildItem extends StatelessWidget {
   const ChildItem({
-    required this.name,
-    required this.total,
-    required this.currencySymbol,
-    required this.pending,
+    required this.profile,
     super.key,
   });
 
-  final String name;
-  final double total;
-  final double pending;
-  final String currencySymbol;
+  final Profile profile;
```

Comment:
Do you really need the whole object? I see you're only using the pending

Reply:
I like such approach. Do you see any issues here?